src/js/components/devices/device-details/authsets/authsets.js

```diff
@@ -37,8 +37,10 @@ export const Authsets = ({ decommission, deleteAuthset, device, getDeviceAuth, l
   };
 
   return (
-    <div style={{ minWidth: 900, marginBottom: theme.spacing(2) }}>
-      {status === DEVICE_STATES.pending ? `Authorization ${pluralize('request', auth_sets.length)}` : 'Authorization sets'}
+    <div style={{ minWidth: 700, marginBottom: theme.spacing(2), backgroundColor: '#f7f7f7', border: '1px solid rgb(224, 224, 224)', padding: '16px' }}>
```

Comment:
just FYI: in order to get to uniform padding/ margin sizes I try to use the `theme.spacing` and import the colors from the theme file - so that combined with the `withStyles` option we could ease the customizability at some point